Lib/symtable.py

```diff
@@ -4,8 +4,11 @@
 from _symtable import (
     USE,
     DEF_GLOBAL, DEF_NONLOCAL, DEF_LOCAL,
-    DEF_PARAM, DEF_TYPE_PARAM, DEF_IMPORT, DEF_BOUND, DEF_ANNOT,
-    SCOPE_OFF, SCOPE_MASK,
```

Comment:
Renaming `SCOPE_OFF` is potentially a breaking change. Maybe add an alias just for the case?

Reply:
Sure! Should I make it deprecated if someone imports it using the `__getattr__` module-level construction:

```python
def __getattr__(name):
    if name == 'SCOPE_OFF':
        import warnings
        warnings.warn(f"The {name!r} attribute is deprecated, use 'SCOPE_OFFSET' instead",
                      DeprecationWarning, stacklevel=2)
        return SCOPE_OFFSET

    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```

Reply:
Since it is not documented, we are not obliged to keep it, so it can be removed at any time. But this can break the user code, and we want to be nice to users. If we are going to deprecate the old name, it is better to have several releases in which both names can be used without triggering a warning or error.

Alternatively, you can just remove this change from this PR. SCOPE_OFF was here for ages, and there is no great benefit in renaming it.

You can re-request whenever you want, but since I reviewed it only several hours ago, there is not much chance that I miss your changes and comments.

Reply:
> But this can break the user code, and we want to be nice to users

Let's go with this mindset. I will also revert the change so that the code would at least be a bit nicer and the PR smaller. It's a bit weird that it was exported as `SCOPE_OFF` but let's keep what has been working for ages.